crates/ironrdp-rdpdr-native/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "ironrdp-rdpdr-native"
+version = "0.1.0"
+readme = "README.md"
+description = "Native RDPDR static channel backend implementations for IronRDP"
+edition.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+authors.workspace = true
+keywords.workspace = true
+categories.workspace = true
+
+[lib]
+doctest = false
+test = false
+
+
+[target.'cfg(any(target_os = "macos", target_os = "linux"))'.dependencies]
+ironrdp-pdu.workspace = true
+ironrdp-svc.workspace = true
+ironrdp-rdpdr.workspace = true
+nix = { version = "0.29", features = ["fs"] }
+libc = "0.2"
```

Comment:
suggestion: You don’t need to specify this dependency at all since it’s re-exported by `nix`.

```suggestion
```